docs/molgenis/run_systemd.md

```diff
@@ -127,12 +127,22 @@ EOL
 <!-- tabs:start -->
 
 #### **Ubuntu (apt)**
+
+```console
+sudo apt update
+```
+
 ```console
 sudo apt install molgenis-emx2 
```

Comment:
this will only install ubuntu packages versions. These are currently not all versions right?
Please make that clear at the beginning of this manual then.